packages/import-handler/src/index.ts

```diff
@@ -223,15 +223,29 @@ const sendSavePageMutation = async (userId: string, input: unknown) => {
   })
 
   const auth = (await signToken({ uid: userId }, JWT_SECRET)) as string
-  const response = await axios.post(`${REST_BACKEND_ENDPOINT}/graphql`, data, {
-    headers: {
-      Cookie: `auth=${auth};`,
-      'Content-Type': 'application/json',
-    },
-  })
+  try {
+    const response = await axios.post(
+      `${REST_BACKEND_ENDPOINT}/graphql`,
+      data,
+      {
+        headers: {
+          Cookie: `auth=${auth};`,
+          'Content-Type': 'application/json',
+        },
+        timeout: 30000, // 30s
```

Comment:
Seems very high but I guess the side effects aren't too bad, it just means the thumbnailer instances stay alive longer and probably slows down thumbnailing and saves the API if things are getting slow.

Reply:
yeah, that's exactly my thought